java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -470,6 +474,27 @@ public static HostColumnVector fromStrings(String... values) {
     });
   }
 
+  /**
+   * Create a new vector from the given values.  This API supports inline nulls, but it is inefficient.
+   * Notice all input BigDecimals should share same scale.
```

Comment:
This needs to document the scale behavior when all the decimal values are null.  Note that the scale of `BigDecimal.ZERO` is 1 not 0, so the scale behavior for all nulls changed with the latest update.  I'm not sure it's critical which scale is chosen as long as it's documented.  The user can call the `build(DType, int, Consumer<Builder>)` API directly if they need to control the scale value when constructing a column vector that may contain all null decimals.

Reply:
I think the scale of `BigDecimal.ZERO` is 0 (the precision is 1). And the lower-level API `build(DType, int, Consumer<Builder>) ` looks good to me.

Reply:
> I think the scale of BigDecimal.ZERO is 0 (the precision is 1).

Ah yes, I accidentally was looking at the precision.  The comment I made still applies:
> This needs to document the scale behavior when all the decimal values are null.

Reply:
It's done.